### ISSUE TITLE
关于“02.recognize_digits”JupyterNotebook版本README.cn的疑问和建议

### ISSUE THREAD

**Issue by JiabinYang** [PaddlePaddle/book#569]
以下问题是以docker run -d -p 8888:8888 paddlepaddle/book启动相应Docker image，进入book后测试过程中发生的，使用过程中希望分别通过CPU和GPU进行训练，但train with GPU未成功。

1. 这里使用net0，net1不太合适，是否使用neural0, neural1更合适
![2018-07-11 5 23 16](https://user-images.githubusercontent.com/22361972/42573271-3439f32c-854e-11e8-9701-b521b1684939.png)

2. 损失交叉函数（cross entropy loss)是否应该应补全
等号左边：Loss(cross entropy)是否更合适
![2018-07-11 5 25 53](https://user-images.githubusercontent.com/22361972/42573867-d7d1a8bc-854f-11e8-91b7-d1d3827070f5.png)

3. Softmax的准确率比叙述低2%，是否影响
![2018-07-11 7 27 21](https://user-images.githubusercontent.com/22361972/42573914-f49f27bc-854f-11e8-977c-54c83ac05071.png)
